packages/provider.multi/spec/MultiProviderBehaviors.ts

```diff
@@ -3,6 +3,8 @@ import {
   MultiProvider
 } from '../dist'
 
+import { assert } from "chai"
```

Comment:
should be before relative imports

Reply:
Done, https://github.com/Auge19/tko/pull/4